TpcProduction/GarfieldProduction/StartScript.sh

```
@@ -0,0 +1,31 @@
+             -v i="$1" \
+'BEGIN {
+   print bmin + (bmax - bmin) * i / nb
+}')
+
+echo $bnow
+
+output="/direct/phenix+u/workarea/hemmick/code.sphenix/tkh/gas/gasfiles/PART_"$1".gas"
+
+echo $output
+
+echo GasModel $emin $emax $ne $bnow $bnow 1 $amin $amax $na $output
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)